mix.lock

```diff
@@ -1,5 +1,4 @@
 %{"bbmustache": {:hex, :bbmustache, "1.0.4", "7ba94f971c5afd7b6617918a4bb74705e36cab36eb84b19b6a1b7ee06427aa38", [:rebar], []},
```

Comment:
What are you using `bbmustache` for? Is that a sub-dependency of something else?

It's interesting because we will actually need to use this! Good stuff. It's already there.